src/actions/breakpoints.js

```diff
@@ -407,6 +408,52 @@ export function toggleBreakpoint(line: ?number, column?: number) {
   };
 }
 
+export function toggleBreakpointsAtLine(line: number, column?: number) {
```

Comment:
do we need the column here: `column?: number`

Reply:
Yes, because this code also *adds* a breakpoint, so it keeps parity with `toggleBreakpoint`; I know we don't use a column breakpoint UI now but we will in the future.